internal/cmd/rename_agent.go

```diff
@@ -0,0 +1,43 @@
+/*
+ *  *******************************************************************************
+ *  * Copyright (c) 2019 Edgeworx, Inc.
+ *  *
+ *  * This program and the accompanying materials are made available under the
+ *  * terms of the Eclipse Public License v. 2.0 which is available at
+ *  * http://www.eclipse.org/legal/epl-2.0
+ *  *
+ *  * SPDX-License-Identifier: EPL-2.0
+ *  *******************************************************************************
+ *
+ */
+
+package cmd
+
+import (
+	rename "github.com/eclipse-iofog/iofogctl/internal/rename/agent"
+	"github.com/eclipse-iofog/iofogctl/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+func newRenameAgentCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "agent NAME NEW_NAME",
+		Short:   "Rename a Agent to another name in your ECN",
+		Long:    `Rename a Agent to another name in your ECN`,
+		Example: `iofogctl rename agent NAME NEW_NAME`,
+		Args:    cobra.ExactArgs(2),
+		Run: func(cmd *cobra.Command, args []string) {
+			// Get name and the new name of agent
+			name := args[0]
+			newname := args[1]
+
+			// Get an executor for the command
+			err := rename.Execute("", name, newname)
```

Comment:
How come "" for namespace parameter?

Reply:
It hows the other executors deal with namespaces, like delete, it should when we do GetNamespace default to current namespace, which is specified and defined under -n "namespace" correct?